### PR TITLE
Docs: Replace Latin in the porting guides

### LINKED ISSUE #79000: Docs: Replace Latin terms in the /porting guides files for English terms
### Summary

Our [style guide](https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html#avoid-using-latin-phrases) notes that we should not use Latin terms (e.g, via, etc) in our documentation. This issue specifically asks to replace these with the noted English equivalents from that style guide table for all occurrences in the docs/docsite/rst/porting_guides/ directory .

Use `grep -R -e 'etc\.' -e 'i\.e ' -e 'e\.g\. ' -e 'via ' -e 'vs\(\.\)\? ' -e versus --exclude=platform* `  in the docs/docsite/rst/porting_guides/ directory to find these.

List of all effected files are in a follow-on comment. 

### Issue Type

Documentation Report

### Component Name

docs/docsite/porting_guides/core_porting_guides.rst

### Ansible Version

```console
$ ansible --version
2.14
```

### Configuration

```console
# if using a version older than ansible-core 2.12 you should omit the '-t all'
$ ansible-config dump --only-changed -t all
none
```

### OS / Environment

none

### Additional Information

none

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by sgaist**
##### SUMMARY

Replace latin terms with english in the scenario_guides directory as per the [style guide](https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html#avoid-using-latin-phrases).

Fixes #79000

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

Documentation: Porting Guides

### CODE REVIEW

File: docs/docsite/rst/porting_guides/porting_guide_2.0.rst [hunk 1]
```diff
@@ -149,7 +149,7 @@ Should now be::
         a: 1
 
 * Setting any_errors_fatal on a task is no longer supported. This should be set at the play level only.
-* Bare variables in the `environment` dictionary (for plays/tasks/etc.) are no longer supported. Variables specified there should use the full variable syntax: '{{foo}}'.
+* Bare variables in the `environment` dictionary (for plays/tasks/and so on) are no longer supported. Variables specified there should use the full variable syntax: '{{foo}}'.
```
Comment (Akasurde) [inline docs/docsite/rst/porting_guides/porting_guide_2.0.rst]:
```suggestion
* Bare variables in the `environment` dictionary (for plays, tasks, and so on) are no longer supported. Variables specified there should use the full variable syntax: '{{foo}}'.
```

File: docs/docsite/rst/porting_guides/porting_guide_2.10.rst [hunk 2]
```diff
@@ -43,8 +43,8 @@ Command Line
 ============
 
 * The ``ansible-galaxy login`` command has been removed, as the underlying API it used for GitHub auth is being shut down. Publishing roles or
-  collections to Galaxy via ``ansible-galaxy`` now requires that a Galaxy API token be passed to the CLI via a token file (default location
-  ``~/.ansible/galaxy_token``) or (insecurely) via the ``--token`` argument to ``ansible-galaxy``.
+  collections to Galaxy through ``ansible-galaxy`` now requires that a Galaxy API token be passed to the CLI through a token file (default location
+  ``~/.ansible/galaxy_token``) or (insecurely) through the ``--token`` argument to ``ansible-galaxy``.
```
Comment (Akasurde) [inline docs/docsite/rst/porting_guides/porting_guide_2.10.rst]:
```suggestion
  ``~/.ansible/galaxy_token``) or (insecurely) using the ``--token`` argument to ``ansible-galaxy``.
```

File: docs/docsite/rst/porting_guides/porting_guide_base_2.10.rst [hunk 3]
```diff
@@ -36,8 +36,8 @@ Command Line
 ============
 
 * The ``ansible-galaxy login`` command has been removed, as the underlying API it used for GitHub auth is being shut down. Publishing roles or
-  collections to Galaxy via ``ansible-galaxy`` now requires that a Galaxy API token be passed to the CLI via a token file (default location
-  ``~/.ansible/galaxy_token``) or (insecurely) via the ``--token`` argument to ``ansible-galaxy``.
+  collections to Galaxy through ``ansible-galaxy`` now requires that a Galaxy API token be passed to the CLI through a token file (default location
+  ``~/.ansible/galaxy_token``) or (insecurely) through the ``--token`` argument to ``ansible-galaxy``.
```
Comment (Akasurde) [inline docs/docsite/rst/porting_guides/porting_guide_base_2.10.rst]:
```suggestion
  ``~/.ansible/galaxy_token``) or (insecurely) using the ``--token`` argument to ``ansible-galaxy``.
```

File: docs/docsite/rst/porting_guides/porting_guide_core_2.12.rst [hunk 4]
```diff
@@ -112,7 +112,7 @@ Plugins
 
 * ``unique`` filter with Jinja2 < 2.10 is case-sensitive and now raise coherently an error if ``case_sensitive=False`` instead of when ``case_sensitive=True``.
 * Set theory filters (``intersect``, ``difference``, ``symmetric_difference`` and ``union``) are now case-sensitive. Explicitly use ``case_sensitive=False`` to keep previous behavior. Note: with Jinja2 < 2.10, the filters were already case-sensitive by default.
-* ``password_hash``` now uses ``passlib`` defaults when an option is unspecified, e.g. ``bcrypt_sha256`` now default to the "2b" format and if the "2a" format is required it must be specified.
+* ``password_hash``` now uses ``passlib`` defaults when an option is unspecified, for example ``bcrypt_sha256``, now default to the "2b" format and if the "2a" format is required it must be specified.
```
Comment (Akasurde) [inline docs/docsite/rst/porting_guides/porting_guide_core_2.12.rst]:
```suggestion
* ``password_hash``` now uses ``passlib`` defaults when an option is unspecified, for example ``bcrypt_sha256``, now defaults to the "2b" format, and if the "2a" format is required it must be specified.
```

**Review by Akasurde — CHANGES_REQUESTED** [review]
Rest LGTM

File: docs/docsite/rst/porting_guides/porting_guide_base_2.10.rst (hunk 3 above)
Comment (sgaist) [reply docs/docsite/rst/porting_guides/porting_guide_base_2.10.rst]: Thanks for the suggestion ! I did follow a bit too much the guideline.

File: docs/docsite/rst/porting_guides/porting_guide_2.10.rst (hunk 2 above)
Comment (sgaist) [reply docs/docsite/rst/porting_guides/porting_guide_2.10.rst]: Done

File: docs/docsite/rst/porting_guides/porting_guide_2.0.rst (hunk 1 above)
Comment (sgaist) [reply docs/docsite/rst/porting_guides/porting_guide_2.0.rst]: Thanks for the suggestion, it reads even better.

**Review by Akasurde — APPROVED** [review]

**Comment by oraNod** [comment]
@sgaist Thanks for the contribution! Hacktoberfest is a busy time and it looks like there is already a PR for this issue.

image

There are plenty more [Hacktoberfest issues](https://github.com/ansible/ansible/labels/hacktoberfest) available and we'd love another contribution. See you in the PR queue!

**Pull request closed without merging**